src/Stock.js

```diff
@@ -43,8 +50,16 @@ class Stock extends React.Component {
         let stockTimeValuesInner = [];
         let stockPriceValuesInner = [];
         let stockVolumeValuesInner = [];
+        let companyNameInner = [];
+        let company_52WeekHighInner = [];
+        let company_52WeekLowInner = [];
+        let companyDescriptionInner = [];
+        let priceToSalesRatioInner = [];
+        let profitMarginInner = [];
+        let RevenuePerShareTTMInner = [];
         let API_Mode = ''
         let API_Link = ''
```

Comment:
Now that you've added a second API call to each instance, this variable could probably be renamed to something more descriptive. Something referencing that this Link is what controls the graph values. (and remember to change other instances of this variable to the new name)

Reply:
Variable has been renamed to specify purpose of API call.